docs/source/documentation.rst

```diff
@@ -404,21 +404,69 @@ Help
 
 If you experience issues during installation and/or use of AxonDeepSeg, you can post a new issue on the `AxonDeepSeg GitHub issues webpage <https://github.com/neuropoly/axondeepseg/issues>`_. We will reply to you as soon as possible.
 
-Manual correction
+Guide for manual labelling
+===============================================================================
```

Comment:
sphinx requirement: underline should be same size as subheading number of characters

Reply:
I found that the requirement is for the underline to be "at least as long as the text" ([ref](https://www.sphinx-doc.org/en/1.8/usage/restructuredtext/basics.html#sections)). I don't mind changing it to the same length but I have to change all the other underlines in the document.

Reply:
Resolved in 5d8d474